src/webui/www/private/scripts/client.js

```diff
@@ -750,6 +750,8 @@ window.addEventListener("DOMContentLoaded", () => {
     let syncMainDataTimeoutID = -1;
     let syncRequestInProgress = false;
     const syncMainData = () => {
+        if (document.hidden)
+            return;
```

Comment:
I guess you try to avoid an extra update, but wouldn't it be better to not start the timer at all?
I.e. to do the check before this line

https://github.com/qbittorrent/qBittorrent/blob/15069b26434948e55c4b64dc215f9e1ee0c44241/src/webui/www/private/scripts/client.js#L957

Reply:
I changed it to this so it would have the same behaviour as the property panel - it doesn't do the extra update if the timer is running when you leave the page.

Reply:
Yeah, makes sense. Disregard my comment then.